webpack/scenes/ContentViews/Details/Filters/CVFilterDetailType.js

```diff
@@ -38,6 +38,12 @@ const CVFilterDetailType = ({
         showAffectedRepos={showAffectedRepos}
         setShowAffectedRepos={setShowAffectedRepos}
       />);
+    case 'erratum':
+      if (rules[0]?.types) {
```

Comment:
in cases of doing th rules[0] 
I'd prefer to see us get in the habit of using lodash (head): 
https://lodash.com/docs/4.17.15#head 

Lodash gives us all the nice ruby methods and more for arrays, which makes things a little cleaner. 

#optional